.cp-remote-ignore

```diff
@@ -0,0 +1,11 @@
+.idea
+.git
+*___jb_old___
+*___jb_tmp___
+/cp-remote-logs**
+/.cp-remote-settings.yml
+/.cp-remote-env-settings.yml
+/.cp-remote-ignore
+
```

Comment:
Could we add `vendor` to this file? With an appropriate .cp-remote-ignore-fetch to allow fetching down the vendor folder?